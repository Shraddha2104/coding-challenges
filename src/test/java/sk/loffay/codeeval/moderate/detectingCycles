12 15 10 0 2 9 6 11 7 8 5 3 18 16 4 17 1 13 14 37 42 37 42 37 42 37 42
2 0 16 19 15 9 6 20 8 7 14 22 13 12 4 5 1 21 10 3 11 17 84 72 18 81 43 41 71 76 70 17 84 72 18 81 43 41 71 76 70 17 84 72 18 81 43 41 71 76 70 17 84 72 18 81 43 41 71 76 70
0 3 15 2 14 21 10 6 7 1 4 19 18 17 16 11 13 20 5 12 27 9 24 97 73 8 52 88 95 27 9 24 97 73 8 52 88 95 27 9 24 97 73 8 52 88 95 27 9 24 97 73 8 52 88 95
21 16 8 42 3 19 37 10 20 39 6 11 17 31 46 47 2 5 38 22 24 7 41 9 18 12 25 4 26 48 1 27 30 0 15 29 40 34 36 44 23 14 13 49 53 51 33 45 28 43 79 35 32 53 51 33 45 28 43 79 35 32 53 51 33 45 28 43 79 35 32 53 51 33 45 28 43 79 35 32
10 15 8 13 6 12 4 14 1 0 5 11 7 9 3 16 2 86 76 82 86 76 82 86 76 82 86 76 82
35 38 43 42 1 21 20 16 19 46 11 13 48 45 47 8 22 9 3 4 12 15 24 36 7 39 32 30 5 33 28 25 34 17 2 10 26 41 18 44 23 31 14 6 29 0 40 27 37 90 90 90 90
24 14 25 1 3 21 7 10 16 27 13 5 28 11 4 18 0 2 9 26 6 23 22 17 20 19 15 8 29 12 29 12 29 12 29 12
39 31 22 0 15 42 24 16 36 8 10 6 38 7 32 37 12 41 2 28 5 33 29 27 35 23 18 43 4 26 21 3 17 11 14 20 40 9 30 13 19 78 25 1 59 83 99 58 75 55 34 78 25 1 59 83 99 58 75 55 34 78 25 1 59 83 99 58 75 55 34 78 25 1 59 83 99 58 75 55 34
27 41 34 40 22 14 31 7 3 11 21 43 20 4 33 42 5 18 44 0 39 12 17 10 37 25 32 16 2 29 8 36 24 30 19 13 15 26 35 38 6 1 9 84 87 23 68 28 84 87 23 68 28 84 87 23 68 28 84 87 23 68 28
45 4 38 18 29 8 33 5 3 17 11 26 40 39 34 37 6 23 10 27 46 2 30 28 16 12 41 24 36 42 21 32 14 25 19 31 22 13 43 1 7 44 35 9 20 15 0 91 57 0 91 57 0 91 57 0 91 57
4 3 1 0 6 7 2 5 20 20 20 20
12 20 1 11 9 2 7 16 5 3 8 18 13 17 10 21 4 0 6 15 14 74 73 95 19 71 74 73 95 19 71 74 73 95 19 71 74 73 95 19 71
9 14 4 20 36 31 3 21 6 24 12 26 2 5 37 15 30 32 29 8 35 11 27 18 13 25 28 19 10 23 22 1 16 0 89 17 33 76 74 38 34 47 7 89 17 33 76 74 38 34 47 7 89 17 33 76 74 38 34 47 7 89 17 33 76 74 38 34 47 7
0 3 2 1 2 1 2 1
1 13 14 18 4 7 0 15 11 2 9 17 10 3 5 12 8 16 6 6 6 6
4 56 56 56 56 56 56 56 56 56 56
4 8 5 9 3 7 1 2 6 53 91 0 47 53 91 0 47 53 91 0 47 53 91 0 47
1 2 3 1 2 3 1 2 3 1 2 3 1 2 3 1 2 3 1 2 3
5 1 2 3 4 0 13 71 66 13 71 66 13 71 66 13 71 66
37 38 0 16 7 22 4 18 15 9 40 27 11 10 12 8 5 41 19 36 32 21 28 20 13 2 34 24 29 17 26 23 31 30 25 1 35 64 45 33 3 6 39 94 93 14 98 64 45 33 3 6 39 94 93 14 98 64 45 33 3 6 39 94 93 14 98 64 45 33 3 6 39 94 93 14 98
