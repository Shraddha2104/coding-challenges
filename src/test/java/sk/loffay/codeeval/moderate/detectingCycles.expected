37 42
17 84 72 18 81 43 41 71 76 70
27 9 24 97 73 8 52 88 95
53 51 33 45 28 43 79 35 32
86 76 82
90
29 12
78 25 1 59 83 99 58 75 55 34
84 87 23 68 28
0 91 57
20
74 73 95 19 71
89 17 33 76 74 38 34 47 7
2 1
6
56
53 91 0 47
1 2 3
13 71 66
64 45 33 3 6 39 94 93 14 98
